bech32/app/Main.hs

```diff
@@ -127,12 +127,12 @@ hrpArgument :: Parser HumanReadablePart
 hrpArgument = argument (eitherReader reader) $ mconcat
     [ metavar "PREFIX"
     , helpDoc $ Just $ vsep
-        [ text "An optional human-readable prefix (e.g. 'addr')."
-        , indent 2 $ text
-            "- When provided, the input text is decoded from various encoding \
+        [ pretty "An optional human-readable prefix (e.g. 'addr')."
+        , indent 2 $ pretty
+            "- When provided, the input pretty is decoded from various encoding \
```

Comment:
Oops! 😄 

I've fixed this in https://github.com/input-output-hk/bech32/pull/60.